tiptapy/__init__.py

```diff
@@ -39,22 +43,37 @@ def _get_abs_template_path(path_str):
     return os.path.join(pkg_dir, path_str)
 
 
-class BaseDoc:
+def escape_recursive(node):
```

Comment:
naming suggestion-> escape_json

Reply:
minor refactor suggestion:

def escape_json(data):
    """ Escape strings, or recursively escape lists and dictionaries.
    skip escape for dictionary keys listed in skip_keys
    """
    skip_keys = ["html"]
    ...

Reply:
On naming: it’s not escaping json. It’s taking dict (block/node) as input and recursively html escaping all the values.

on skip_keys, it’s currently only one attribute we need to skip. Not sure if there will ever be another key to skip. So making a separate list for it seems overkill for now.

Reply:
yeah any name works, escape_recursive is also ok, in my mind  escape_value or escape_json sounds better..

"html" in if condition is without any context and is confusing, if not list, can be assigned to a variable at the least.
and a comment can be added

Reply:
Renamed to `escape_values_recursive`. Also added `skip_key = "html"  # Skip escaping html values in embeds`.